sms-gateway.php

```diff
@@ -26,7 +26,10 @@
 <?php   }
     } 
     else if (str_exists(strtoupper($address), strtoupper('jft'))) {
-        $message = $client->messages->create($_REQUEST['From'], array("from" => $_REQUEST['To'], "body" => get_jft(true)));
+        $message_get = get_jft(true);
```

Comment:
let's change the name of this array to something more meaningful `$jft_chunks`, or something similar.

Reply:
word thanks, when i named that i knew it was aweful